### ISSUE TITLE
[LiveComponent] please give full examples

### ISSUE THREAD

**Issue by robert-hoffmann** [symfony/ux#655]
This seems interesting

But your example here
https://ux.symfony.com/live-component#demo

Uses a loop to show items (ok, i got this working)

But your example here
https://ux.symfony.com/live-component/demos/voting

Simply shows a template snippet with zero looping ..so how it this working ?

because when i try to implement it, i get
`An exception has been thrown during the rendering of a template ("The property "App\Components\LiveUserComponent::$user" is not readable because it is typed "App\Entity\User". You should initialize it or declare a default value instead.").`

I've used the exact same entity in both examples i tried to implement

Would strongly appreciate full source code to the examples shown on ux.symfony.com
It's already complicated enough that example code, and documentation code do not align and need adaptation ;-)

Thank you !

**Comment by wouterj** [comment]
The code shows the `food_vote` component source. Each row is its own component "instance", which is rendered in a loop.

The full source is in this repository (in the `ux.symfony.com/` directory). E.g. the loop you're looking for is: https://github.com/symfony/ux/blob/2.x/ux.symfony.com/templates/live_component_demo/voting.html.twig

But I'm not involved much in the UX initiative, so I let others decide on what to improve in the rendered example/docs here :)

**Comment by robert-hoffmann** [comment]
Thanks, that makes it a little easier to navigate the examples !

Should probably link this on the ux.symfony.com page with the examples, as most ppl (me included) will just copy-paste the example code (with a few adaptations) and expect it to work ;-)

Also maybe update the code, for example
src/Twig/FoodVoteComponent.php

is now located in
src/Components/FoodVoteComponent.php

Ideally, i'd suggest making a full blown symfony project with ALL the examples (in working condition), and just allow users to download that and play with it. This could also double as as test ground for the devs, to make sure their examples actually still work, as progress on the code goes forward.

Ok take that back, found it ;-)
https://github.com/symfony/ux/tree/2.x/ux.symfony.com
(but seems to need updating to reflect the new code changes)

**Comment by weaverryan** [comment]
Hi!

> Ideally, i'd suggest making a full blown symfony project with ALL the examples (in working condition), and just allow users to download that and play with it. This could also double as as test ground for the devs, to make sure their examples actually still work, as progress on the code goes forward.

That is kind of what I was thinking for ux.symfony.com - it's a site to demo things, but you can also use it locally. 

However, I can see 2 problems immediately:

A) The `ux.symfony.com` sub-directory doesn't have a README about how to set it up. Plus, we don't link or mention it anywhere as a demo project.

B) On https://ux.symfony.com/live-component/demos/voting, that's a really good point that we're zooming in too much, and so you can't really see the full picture. The full picture would include:

1) The original controller https://github.com/symfony/ux/blob/2.x/ux.symfony.com/src/Controller/LiveComponentDemoController.php#L61-L68
2) The template that renders the component https://github.com/symfony/ux/blob/2.x/ux.symfony.com/templates/live_component_demo/voting.html.twig#L20-L26
3) Then also the component class and its template

Basically, I think we should have some sort of a "See the code" link somewhere on that page that takes you directly to GH so you can browse around. Maybe a "See the code" link directly below the 2 code blocks. On click, it rolls down to show a list of files, each which is a link to GH.

Thanks for the convo - ux.symfony.com is a great demo resource... let's leverage it better!

**Comment by robert-hoffmann** [comment]
Maybe you could move the site to a different repository and like that it would be easier to clone.

Also might make updating it simpler ? As of now it's a few versions behind the "semi-official" docs on symfony.com

**Comment by weaverryan** [comment]
> Maybe you could move the site to a different repository and like that it would be easier to clone.

That's a pretty good idea - it would increase visibility as well.
